plutus-benchmark/marlowe/src/PlutusBenchmark/Marlowe/Scripts/Semantics.hs

```diff
@@ -111,6 +122,15 @@ closeInterval (Interval (LowerBound (Finite (POSIXTime l)) lc) (UpperBound (Fini
     )
 closeInterval _ = Nothing
 
+-- TODO: abstract away
+toDataValue :: SOP.Value -> Data.Value
```

Comment:
This could be another reason for the much worse numbers, so again I'd suggest continuing to use the SOP value for now and change it later.